io/teca_cf_reader.cxx

```diff
@@ -31,7 +31,6 @@ using std::cerr;
 #include <openssl/sha.h>
 #endif
 
-// macro to help with netcdf data types
```

Comment:
don't remove comment

Reply:
I don't remember removing any comment. you are right, they shouldn't be removed.